src/context/WalletProvider/NewWalletViews/NewWalletViewsSwitch.tsx

```diff
@@ -181,12 +184,17 @@ export const NewWalletViewsSwitch = () => {
           onWalletSelect={handleWalletSelect}
         />
         <Divider mb={6} />
+        <HardwareWalletsSection
+          selectedWalletId={selectedWalletId}
+          onWalletSelect={handleWalletSelect}
+          isLoading={isLoading}
+        />
+        <Divider mb={6} />
```

Comment:
This divider size feels a bit excessive:

<img width="207" alt="Screenshot 2025-01-21 at 16 11 09" src="https://github.com/user-attachments/assets/4fb764ea-76c4-4095-bf0f-beedcd955cf2" />

Reply:
smolish margins ftw! 647fa62067
<img width="359" alt="image" src="https://github.com/user-attachments/assets/dcafc675-fb23-4b43-8dde-8ddc720efda7" />